recipes/python-wand/LICENSE

```diff
@@ -0,0 +1,20 @@
+Original work Copyright (C) 2011-2018 by Hong Minhee <https://hongminhee.org>
```

Comment:
Remove this file as it is in the source tarball

Reply:
Done